Automattic/jetpack
# Boost: Refresh scores on theme change

Fixes #

#### Changes proposed in this Pull Request:
If the theme has changed since the last time speed score was measured, load fresh scores.

#### Jetpack product discussion
No

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Go to Boost option page and make sure the scores are loaded.
* Change the theme
* Come back to Boost option page
* Now you should see one of the following:
    1. If Critical CSS is inactive, speed score will first load with the previous value and instantly start loading fresh.
    2. If Critical CSS is active, previous speed score will be loaded and it will wait for critical CSS gen to finish before fetching fresh.
